opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -244,16 +245,42 @@ message Span {
   }
 
   // The included links.
-  Links links = 12;
+  Links links = 13;
 
   // An optional final status for this span. Semantically when Status
   // wasn't set it is means span ended without errors and assume
   // Status.Ok (code = 0).
-  Status status = 13;
+  Status status = 14;
 
   // An optional number of child spans that were generated while this span
   // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 14;
+  google.protobuf.UInt32Value child_span_count = 15;
+}
+
+// AttributeKeyValue is a key-value pair that is used to store Span attributes, Link
+// attributes, etc.
+message AttributeKeyValue {
+  // ValueType is the enumeration of possible types that value can have.
+  enum ValueType {
+    STRING  = 0;
+    INT64   = 1;
```

Comment:
nit: Do we need the 64 specifier? Before it was simply int.

Reply:
I agree. Changed to simply `INT`.